release/release.sh

```diff
@@ -1,21 +1,38 @@
 #!/bin/bash
 #
 # Usage:
-#   $ release.sh [-f] <organization/project-name>
+#   $ release.sh [-f] <organization/project-name> [new-version]
+#
+#   Args:
+#     organization/project-name    Required. The GitHub repo to release.
+#
+#     new-version                  Optional. The new version number to use,
+#                                  specified as M.N.0. If not specified, the
+#                                  new version will be computed from existing
+#                                  git tags. This flag should only be needed
+#                                  when jumping to a non-sequential version
+#                                  number.
 #
 #   Options:
 #     -f     Force; actually make and push the changes
 #     -h     Print help message
 #
 # Example:
-#   # Shows what commands would be run. NO CHANGES ARE PUSHED
+#   # NO CHANGES ARE PUSHED. Shows what commands would be run.
 #   $ release.sh googleapis/google-cloud-cpp-spanner
 #
-#   # Shows commands AND PUSHES changes when -f is specified
+#   # NO CHANGES ARE PUSHED. Shows what commands would be run.
+#   $ release.sh googleapis/google-cloud-cpp-spanner 2.0.0
+#
+#   # PUSHES CHANGES to release -spanner
 #   $ release.sh -f googleapis/google-cloud-cpp-spanner
 #
+#   # PUSHES CHANGES to release -spanner, setting its new version to 2.0.0
+#   $ release.sh -f googleapis/google-cloud-cpp-spanner 2.0.0
+#
 # This script creates a "release" on github by doing the following:
```

Comment:
Perhaps this should come before the examples?

Reply:
Done.